.dockerignore

```diff
@@ -1,6 +1,9 @@
 ### IDE files
 .idea/
 
+### Composer files
+auth.json
```

Comment:
it will also break dev usage of composer _(with dev config mounting folder, afaik mount also takes entries here into account)_

With multi stage we can safely remove it in build stage anyway right?